### ISSUE TITLE
Sprint 9 (Part 1): Individual Videos and Scores

### ISSUE THREAD

**Issue by rebecca-123** [Rebecca-123/m223_team_lovelace#74]
**[More Organized Google Doc Version](https://docs.google.com/document/d/1JRH_MvMWFynRXpHhN-IO5GQ2-EJ2fVUPRyWw75sTYwY/edit)**

**[Christina’s Video](https://youtu.be/V-jZTAIZ1uQ)**
**Average Score: 5.73/6**

| Team Member  | Score                                        | Positives                                            | Improvement                       |
| :---------------:  | :---------------------------------| :--------------------------------------- | :-----------------------------|
| Rebecca            |5.45/6, -0.3 (.15 * 2) Connection to College Board topics was not very clear, didn’t cite topic numbers, - 0.25 Try to add explanation of RGB code (ie brief Tour of Code)| 1. Nice demo and explanation of code 2. Organized issues with checklist 3. Shows command of Logic Gate technical | More thorough connection between technical and College Board topic, thought the explanation for variables was too brief
| Ritvik                | 6/6                                             | 1. Good use of issues to show different topics clearly.   2. Good organization of code, specifically citing topics.   3.Perfect timing of video. | Binary math conversions page could be changed to a lighter background, making it more visible to the viewer. | 
| William             | 5.75/6, -0.25 connection to collegeboard was not clear in Variables and Assignments | 1. In depth explanation of code 2. Informative demonstration of website 3. Logic gates page was well designed| Work on video structure. It was kind of all over the place | 

**[Rebecca’s Video](https://drive.google.com/file/d/1YV7YxSV2fHaXqhzL50q5jzrs4cdFALJW/view)**
**Average Score: 5.87/6**

| Team Member  | Score                                          | Positives                                            | Improvement                       |
| :---------------:  | :----------------------------------| :--------------------------------------- | :-----------------------------|
| Christina           | 5.8/6, -a little bit over time, -command of tour of code a little bit fuzzy | 1. Clear explanation of the unsigned addition with pictures 2. Wonderful tying back to the college board questions 3. Especially loved the slides and brainwrite demonstration for the first topic                                                 | The music was kind of distracting for me. The transition to the middle topic was a bit quick for me. (Just like for some things, the transition was a bit fast and I wasn’t able to catch what you were trying to say). Basically, some transitions were kind of sudden and hard to follow along (creating a kind of a rushed vibe)|
| Ritvik                | 5.9/6, -0.1 over time 15 seconds |1. The editing in this video helped me visually. 2. Good elaboration of collaboration topic, using brainwrite as an example. 3. Good showing of code in IntelliJ. Useful to understand what exactly is going on in the project. | Time wasn’t a major issue, but try to stick inside the 3:15 slot (this was an issue for me too).| 
| William             | 5.9/6, -0.1 went overtime|     1. Presentation was well prepared and are useful visuals  2. Video was well edited and put together 3. Explanations were in-depth and I got a good understanding of how code and website worked. | Half the time you sound like you were whispering. Try to speak up to sound more confident! |                 

**[Ritvik’s Video](https://watch.screencastify.com/v/ntiz1KSHpgAuC7cIS28W)**
**Average Score: 5.2/6**

| Team Member  | Score                                                | Positives                                            | Improvement                       |
| :---------------:  | :--------------------------------------| :--------------------------------------- | :-----------------------------|
| Christina            | 5.75/6: -.25 little bit over time, intro is a bit unclear and rushed in my opinion, the links on your about page was hard to read | 1. Nice use of javascript and organized and clear language use. 2. Great descriptive explanation of the code, even said example inputs and outputs! 3. Playful and fun edition of the code is really interesting! Thanks for the wonderful presentation. | I see that you included the topic names and numbers in the task description, but I feel like it would be better to include tangibles(steps) and links on your process of making the code. I also noticed in the beginning when you were showing the different tasks to keep track of your work, I think it would be better if you could show them in the order that you do them to make it a little easier to follow along with which topic you are talking about.  |
| Rebecca            | 5.45/6, -0.45 (.15 * 3) Connection to College Board was not very clear, -0.1 (17 seconds long)| 1. All 3 technicals are connected to a College Board topic (mentioned)   2. Nice implementation of alerts which allow for user interaction with the website   3. Thorough explanation of each of the 3 technicals  | More advanced/developed technicals, stronger connection to College Board topics (maybe incorporating quiz questions, or being more thorough ie including MOD, booleans)                       | 
| William             | 4.4/6, -1.5 Didn’t connect the topics to PBL. Didn’t demonstrate any relation to the website project. -0.1 overtime|1. Presentation is well organized 2. Well rehearsed, didn’t seem to stutter or pause a lot like I did 3. Code was simple and easy to understand. | Try to explain less of how the code works and what you did overall, and more about how the CBL factors into your code. What do they accomplish? | 

**[William’s Video](https://drive.google.com/file/d/1ij4Qhts_j4CKYL7V5dziKemyPV7wKhkH/view?usp=sharing)**
**Average Score: 5.42/6**

| Team Member  | Score                                                | Positives                                            | Improvement                       |
| :---------------:  | :--------------------------------------| :------------------------------------ | :-----------------------------|
| Christina           | 5.5/6, + relation, + Quiz questions not cited(-.25), Video was kind of long and did not show engaging excitement for the topic in my opinion(-.25) |  Love explanation of the three topics: sounds very experienced on what you were saying. The tasks that you choose for the topics were so great and unique: Amazing to see such demonstration of your coding skills. Your added feature of making the added info hidden really makes me think and ponder how to save space and keep the design simple so that it is user friendly.| I would like it if you would create a shorter video. Explanation of the code was kind of confusing, especially for the last topic, iterations, you keeps scrolling throughout the code making your words hard to keep track of what you were saying. I would recommend demonstrating through snippets of code in slides or something like that. |
| Rebecca            |5.5/6, -0.5 (0.1 * 5), video is 4:35| 1.College Board topics clearly addressed 2.Nice use of hidden attribute for planets page 3. Black hole page and mass comparison to sun is really cool| For black holes dropdown, maybe it’d be easier if the dropdown went upwards so the user doesn’t have to scroll down. Other minor suggestion: if possible keep variable names lowercase | 
| Ritvik                |5.25/6, - 0.25 PBL connection could be better for nested conditionals, but is ok. - 0.50 way overtime (0.1 * 5 for 80 seconds over)| 1. Planets page is well done and easy to view. 2. Code is properly organized. 3. Black hole page looks good| You did not give quiz questions to support, which was also a flaw of my video. Also, a conditional could also be finding a value as false, doesn’t always have to be true. I feel like Nested Conditionals (3.7) and Conditionals (3.6) are basically the same thing, and you could choose a different topic to make it more unique for the viewer. You could also try to work on speaking more clearly, without any pauses. I don’t really understand the black hole page. What is the point of displaying the suns? How does that affect our website? |

**Comment by AkhilNandhakumar** [comment]
### Akhil's Overall Score of Ritvik: **7.25/8**

**Video Topic Scores:**
Mathematical Expressions (3.3): Runtime was very clear and and well paced. The code explanation was also very thorough and connected to the college-board topic perfectly. Code/examples did not fully connect to the PBL theme/website purpose. **1.75/2**

Variables and Assignments (3.1): The runtime was again clear and the code tour was also well paced and explained well. The college-board relevance was perfect and easy to understand. Again the examples were not PBL related or showed meaning/purpose that fit with the website theme. **1.75/2**

Conditionals (3.6): The runtime was great, well paced, easy to understand. Connection to college-board topic is perfect, but again it lacked relevance to your PBL/website theme/purpose. **1.75/2**

**Glows:**
The pacing of the video was really good. When you were explaining the code and the runtime, your timing was on point and it was the technical terms you were using were great. I liked the small games/interactive elements you made that was engaging for the user and also connecting to the college-board topics and showed understanding. The way you structured the video with a clear intro, lead in to each section, and the conclusion was nice.

**Grows:**
The only thing that lost you points for the video was the PBL relevance. I like that you added each of the interactive games on your about me page, but it would have been even better to incorporate the college board topics into code that was purposeful for the entire website; maybe an interactive guesser for the user to guess the names of planets, comets, nebulas, galaxies, etc. instead of a simple number guesser.

Comment score: **2/2**
you referenced specific parts of the other team members' videos, gave insightful feedback and graded appropriately. Great job overall!
